google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/stub/readrows/BigQueryStorageException.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.bigquery.storage.v1.stub.readrows;
+
+import com.google.cloud.grpc.BaseGrpcServiceException;
+
+public class BigQueryStorageException extends BaseGrpcServiceException {
```

Comment:
Am I right in understanding that exceptions which are visible to end users today are all BaseGrpcServiceExceptions, and so this should prevent this change from being a breaking change from a user perspective? I'm wondering about the case where a caller has already implemented their own special-case handling for e.g. exceptions produced on INTERNAL error responses from the server.